docs/learn/learn-safrole.md

```diff
@@ -0,0 +1,33 @@
+---
+id: learn-safrole
+title: "Polkadot Block Production: SAFROLE"
+sidebar_label: SAFROLE
+description: The Consensus Mechanism of Polkadot.
+keywords:
+  [
+    consensus,
+    proof of stake,
+    nominated proof of stake,
+    hybrid consensus,
+    block production,
+    sassafras,
+    safrole,
+    babe,
+  ]
+slug: ../learn-safrole
+---
+
+> This page will be updated as progress ensues.
+
+SAFROLE is a SNARK-based block production algorithm. It uses SNARK specifically for their anonymity
+features. And it delivers constant time block production, almost entirely fork-free. There are a
```

Comment:
I would add the "why" here. Afaik anonymity is used to protect block producers' identity from being used to attack

Reply:
Added quite a few details, please see if it makes sense!